src/c/project-basics-edit.js

```diff
@@ -70,6 +69,11 @@ const projectBasicsEdit = {
         vm.loadCategoriesOptionsTo(categories, vm.fields.category_id());
 
         const addTag = tag => () => {
+            if (selectedTags().length >= 5) {
```

Comment:
we should be able to have up to five tags. when trying to add the 6th tag we disallow the user, and at this moment the length of this array is still 5.